tests/test_mask.py

```diff
@@ -42,13 +42,13 @@ def setup_class(cls):
         cls.maskEstimator = cls.faceEngine.createMaskEstimator()
 
         cls.medicalMaskWarpNProperties = WarpNExpectedProperties(
-            FaceWarpedImage(VLImage.load(filename=FACE_WITH_MASK)), MaskProperties(0.000, 0.999, 0.000)
+            FaceWarpedImage(VLImage.load(filename=FACE_WITH_MASK)), MaskProperties(0.000, 0.884, 0.112)
         )
         cls.missingMaskWarpNProperties = WarpNExpectedProperties(
-            FaceWarpedImage(VLImage.load(filename=WARP_CLEAN_FACE)), MaskProperties(0.998, 0.002, 0.000)
+            FaceWarpedImage(VLImage.load(filename=WARP_CLEAN_FACE)), MaskProperties(0.821, 0.002, 0.178)
         )
         cls.occludedMaskWarpNProperties = WarpNExpectedProperties(
-            FaceWarpedImage(VLImage.load(filename=OCCLUDED_FACE)), MaskProperties(0.260, 0.669, 0.071)  # TODO: bug
+            FaceWarpedImage(VLImage.load(filename=OCCLUDED_FACE)), MaskProperties(0.409, 0.018, 0.572)  # TODO: bug
```

Comment:
Если результирующая оценка - "перекрыто", то коммент про баг не актуален
```suggestion
            FaceWarpedImage(VLImage.load(filename=OCCLUDED_FACE)), MaskProperties(0.409, 0.018, 0.572)
```